eve/main.yml

```diff
@@ -55,3 +77,28 @@ stages:
             - 'pmd-lint/*.html' # this will create links to every .html files
             - 'spotbugs/*.html'
             - 'code-coverage/*.html'
+      - TriggerStages:
+          stage_names:
+            - docker-build
```

Comment:
Question for you, so here you are doing the following pipeline:
```
└── test # I test my component and only when I finished testing it
    └── docker-build # I start building the image
```
Where you could do:
```
├── test # I test my component
└── docker-build # At the same time I build my docker image
```

Do you want to continue with the first option? or do you want me to suggest how you could move to option 2?

Reply:
@tcarmet here I want to build and push the docker image only after I do the testing. If my tests fail then no docker image. Please let me know if anything we can do parallel in this approach

Reply:
On the `gradle build` step, add the following option `haltOnFailure: true`. So that the pipeline will achieve what you desire.

Reply:
Done. Thank you @tcarmet